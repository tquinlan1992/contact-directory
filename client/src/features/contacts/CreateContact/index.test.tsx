import Enzyme, { shallow } from "enzyme";
import { CreateContact } from ".";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
Enzyme.configure({ adapter: new Adapter() });

describe("CreateContact", () => {
  it("should render the name, email, address input fields", () => {
    const wrapper = shallow(<CreateContact />);
    expect(wrapper).toMatchSnapshot();
  });
});
