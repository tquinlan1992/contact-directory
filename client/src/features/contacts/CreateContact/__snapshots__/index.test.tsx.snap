// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateContact should render the name, email, address input fields 1`] = `
<Form>
  <TextInput
    label="Name"
    onChange={[Function]}
    value=""
  />
  <TextInput
    label="Email"
    onChange={[Function]}
    value=""
  />
  <TextInput
    label="Address"
    onChange={[Function]}
    value=""
  />
  <FormButton
    onClick={[Function]}
  >
    Create Contact
  </FormButton>
</Form>
`;
